dbos/_queue.py

```diff
@@ -66,6 +69,14 @@ def enqueue(
     ) -> "WorkflowHandle[R]":
         from ._dbos import _get_dbos_instance
 
+        context = get_local_dbos_context()
+        if (
+            context is not None
+            and context.priority is not None
+            and not self.priority_enabled
+        ):
+            dbos_logger.warning(f"Priority is not enabled for queue {self.name}")
```

Comment:
Be more specific about what will happen.
```suggestion
            dbos_logger.warning(f"Priority is not enabled for queue {self.name}. Setting priority will not have any effect.")
```